packages/ckeditor5-list/src/listproperties/ui/listpropertiesview.ts

```diff
@@ -98,7 +99,7 @@ export default class ListPropertiesView extends View {
 	/**
 	 * A collection of views that can be focused in the properties view.
 	 */
-	public readonly focusables: ViewCollection = new ViewCollection();
+	public readonly focusables: ViewCollection<FocusableView> = new ViewCollection();
```

Comment:
FYI this could be simplified (here and in other places):
```suggestion
	public readonly focusables = new ViewCollection<FocusableView>();
```